### ISSUE TITLE
Different lists (bullets, checks etc.)?

### ISSUE THREAD

**Issue by hjalmers** [sebgroup/design-library#8]
Currently we only have this defined in design library:
https://designlibrary.sebgroup.com/components/list/

In my opinion that's more of a grid or table, although the styling can be applied to a list too (should probably follow the style and design of a normal table). Nevertheless we need more list types.

**Check lists**
To accommodate for [this use case](https://app.goabstract.com/projects/06ac4a60-17c9-11e8-ae45-d1e9ab8da527/branches/master/commits/bb180f6ffa13cd50ef20aba7153a2f75d9d4a8aa/files/1E9F4DA6-4A6C-4767-9DEC-F80E609E1B56/layers/337984DA-628D-4C9A-BC72-6DB195C569BB) we've added a check list too.

**Bullet lists**
Here's another use case from another project which requires "normal" bullets:
![image](https://user-images.githubusercontent.com/1632755/49634440-fad2e480-f9fc-11e8-8a1e-169025612227.png)

**Examples**
https://stackblitz.com/edit/list-options?embed=1&file=src/app/app.component.html&view=preview

**Comment by JessiNygrenWalhed** [comment]
@splashdust Kan man få bilder? Kommer inte åt länkarna.

**Comment by splashdust** [comment]
@JessiNygrenWalhed 

Checklist:
![image](https://user-images.githubusercontent.com/428824/50096078-cb975100-020e-11e9-9992-6d3792f4fe5c.png)

Ordered list:
![image](https://user-images.githubusercontent.com/428824/50096146-eff32d80-020e-11e9-9437-6b1b163b7cbe.png)

**Comment by hjalmers** [comment]
I've added a few more styles just to have something to start with [here](https://sebgroup.github.io/bootstrap/list-group).

Ordered list is probably the one with highest priority as teams are starting to implement it and all implementations differ a bit, especially the use of colors. I think it would make sense to use the same colors for the following states:

* default state
* active state (list item represents active/current state in a flow i.e. using steps)
* disabled state (list item is not active)

For a list were users can interact with the items, in addition to the above we should also include states for:

* hover
* click/press (active state in a browser)
* focus (might be nice to for lists with tab support)

What do you guys think @splashdust @JessiNygrenWalhed?

**Comment by splashdust** [comment]
Yes, it makes sense set a standard for state-color mapping and interactive behavior. Maybe even define a few common use cases, and define how ordered lists should behave in those cases. Lists used for similar purposes in different apps should behave in similar ways.

I think different considerations apply if the list is not interactive however, like in the PWS example. In that case it is just a style that can be applied in a rich text editor, and all the items will have the same static color. In that case I think it enough that the numbers are in a colored square, that different colors aren't mixed in the same list, that the contrast between number and background is good, and so on.

@hjalmers @JessiNygrenWalhed

**Comment by JessiNygrenWalhed** [comment]
I would like to make a few sketches. Need to see some more examples. Try out some colours and sizes.

**Comment by hjalmers** [comment]
If you want, you can take a look at two pretty common uses cases [here](https://sebgroup.github.io/bootstrap/list-group) (at the end), personally adding more colors would make sense for highlighting that there's an error for a step using red (instead of the green), and yellow for pending, i.e. we're waiting for something before we can tell you if something went wrong (red) or if it's okay (green). Note sure what purple could be used for, perhaps steps yet to be completed if we don't want to use the disabled state (greyed out)? Just some ideas that I think would be useful:)

**Comment by ulde01** [comment]
I think the way SEB Bootstrap is, is good. Personally I like rounded bullets but other than that - thumbs up!
